common/djangoapps/student/views.py

```diff
@@ -614,8 +619,15 @@ def dashboard(request):
     # enrollments, because it could have been a data push snafu.
     course_enrollments = list(get_course_enrollments(user, course_org_filter, org_filter_out_set))
 
-    # sort the enrollment pairs by the enrollment date
-    course_enrollments.sort(key=lambda x: x.created, reverse=True)
+    if configuration_helpers.get_value(
+        "ENABLE_COURSE_SORTING_BY_COURSE_ID",
+        settings.FEATURES["ENABLE_COURSE_SORTING_BY_COURSE_ID"],
```

Comment:
ditto

Reply:
This key has "True"  as default value in "lms/envs/common.py" (https://github.com/raccoongang/edx-platform/pull/1177/files#diff-82bda515879c96ab0641bfcbe58631c3R257).